### ISSUE TITLE
Please mark this repository as discontinued, since dysfunctional

### ISSUE THREAD

**Issue by smileBeda** [mylar3/mylar3#1574]
There is not much that works in Mylar3 :D

Would the maintainers be so kind and save future users/contributors the time, and mark this repo as archived/discontinued?
I mean _no offense_, it's just that the application is truly not functional.

The simplest "search" ends in WPOD's, and navigating the UI is so very much not *arr, that once one figured out how the setting are to be manipulated, it will still not work, even if details are accurate (such as Newznab, working just fine in any actual *arr app, failing connection in Mylar3, same for downloader NZBGet...)

This is a bit frustrating for people who try it anew, and it would probably help to just mention that its not maintained/not functional on the very readme page.

**Comment by mannionp** [comment]
Oh but it is very functional, once you understand how it works and have it set up for your services. It takes time to implement successfully yes, and some aspects could be better sure - but it’s far from dysfunctional.

it also has nothing to do with the *arrs, which you would have realized if you had spent any significant time with it.

**Comment by barbequesauce** [comment]
Oh but it is very functional, once you understand how it works and have it set up for your services. It takes time to implement successfully yes, and some aspects could be better sure - but it’s far from dysfunctional.

it also has nothing to do with the *arrs, which you would have realized if you had spent any significant time with it.

**Comment by Cducharme84** [comment]
It is maintained, the very maintainer often helps people out on the discord but hasn't been able to do a public release since March. But there's many projects without release for that amount of time without a release.

For a non-functional program Mylar sure manages my library well. Almost as if like BBQ sauce said it's likely that you didn't spend the time to make sure your services mylar is reliant on were setup correctly.

This is a you failing, not a mylar failing, so no this will not shuttered and your suggestion will be promptly laughed at.

**Comment by smileBeda** [comment]
I have set this up using Docker.
At first, I would have assumed to connect the Newznab - which has no clear inputs for API _key_ or API _url_, so I had to experiment a bit (they seem called "host" and just "API"). None of the trials worked, the test fails in an error 500 logged in the console. There's no feedback on the screen. Note, I am running a number of (arr) apps with the exact same creds, so I am sure they work.

I also have a local NZBGet, which does work flawlessly with (arr) apps. Similarly here, error 500 in the console logged on "test", no feedback on screen, and yes my credentials are OK.

So I tried to just start running a simple search in the app. Top left, search input, typed a word and hit enter: white page of death. No feedback, the UI gone. I then found that I have to click the button - not hit enter - that somehow shows the "results", although, if I got it right, I would need another API key to use that search successfully. So I used the "weekly" list, and clicked randomly on one of them (add). This added a item in the list called something like (sorry, I have no source rn anymore to copy paste) "Unable to fetch..." in the column of the "name", instead of the actual comic name.

At this point I looked at the commit history of the repo and saw that the last commit was quite a while ago. I looked it up in google, and some Reddit posts appeared sharing similar experiences, and them all waiting for a new tool called "Kapowarr", which however seems to be very much in the alpha phase still.

So - perhaps sorry if I was over-fast with assuming it to be dead, it certainly does not seem to be working well on my end. Also, this app might not be an *arr (hell no, it is not as I found out), but it _is_ generally what is "recommended" when one searches for "the arr for comics" :)
Thus my assumption to find something like an arr. Which I got it - it is not.

It is also not working (for me and many other users)... So I do happily retire the request to mark it abandoned, seems it is not!
_Perhaps_, given I am _not alone_ with the issues (and sorry if you do not believe it, I have 15 years of programming on my buckle) it could deserve a few tiny updates on the UI to make the currently not so obvious (errors and inputs) more obvious to "less tech literate friends" :)

Just a suggestion, at this point.
